### DISCUSSION TITLE
New AMD GPU Fan Control

### DISCUSSION THREAD

**Discussion by reypazz** [Rem0o/FanControl.Releases#1070] [Ideas]
Not sure if this will be truly helpful or not, but after seeing your FAQ entry about new AMD GPUs not being supported..

> Q: My new AMD card can't be controlled, why?
> A: Newer AMD cards are not supported. If you can provide a working sample of code that works with these newer cards, please share!

Radeon Profile (https://github.com/marazmista/radeon-profile) seems to have working controls for new series AMD GPUs.

**Comment by Rem0o** [comment]
Isn't that Linux?

**Reply by reypazz** [reply]
Sure, but still open source code that can control new AMD GPUs. I can delete if you don't find it's worthwhile info.

**Reply by Rem0o** [reply]
In the context of a Windows app, it's irrelevant.
